Tests/Shared/ChannelId.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright © 2020 Stream.io Inc. All rights reserved.
+// Copyright © 2021 Stream.io Inc. All rights reserved.
```

Comment:
Why is this here :(

Reply:
The file was moved and the formatter updates the header automatically. What's wrong with it? :)

Reply:
Ok it was moved, It was suspicious that it remained untouched and only header changed :)